backend/python/app/static/openapi.json

```diff
@@ -339,7 +339,7 @@
         ],
         "description": "Delete an entity based on its ID",
         "responses": {
-          "204": {
+          "200": {
```

Comment:
Maybe we should change delete entity endpoint entity_routes.py to return 204 instead of changing this, as that's what Python user/auth and Typescript user/auth/entity delete endpoints return

Reply:
Yeah we are inconsistent on whether we return content or not from DELETE endpoints across the different endpoints. It's pretty typical to not return anything in REST APIs so nothing wrong with changing this to be a 204, but I've read that it's best practice to always return something from GraphQL APIs, so for consistency I think we may actually change all REST endpoints to return something (the deleted id) as well.

Let's merge this change so at least the documentation reflects the actual API behaviour right now, but we can have a discussion sometime about standardizing API return types/statuses